### ISSUE TITLE
earthlike vs earth-like

### ISSUE THREAD

**Issue by MCMic** [smcameron/space-nerds-in-space#184]
I go that in my server log:
```
snis_server:snis_server.c:12373: Unexpected planet type 'earth-like'
snis_server:snis_server.c:12373: Unexpected planet type 'earth-like'
```

And doing some grep it does seem like there is earthlike in some places and earth-like in some others, but I can’t be sure where is the problem exactly and if some of these are legitimate or if all should use the same phrasing.

**Comment by smcameron** [comment]
I believe it's coming from these lines in the solarsystem files:

```
scameron@wombat ~/github/space-nerds-in-space $ grep earth-like share/snis/solarsystems/*/assets.txt
share/snis/solarsystems/default/assets.txt:planet texture: ../../textures/planet-texture4- no-normal-map earth-like
share/snis/solarsystems/default/assets.txt:planet texture: ../../textures/planet-texture4- no-normal-map earth-like
share/snis/solarsystems/polaris/assets.txt:planet texture: zebina-with-clouds- no-normal-map earth-like
share/snis/solarsystems/sirius/assets.txt:planet texture: ourck-with-clouds- no-normal-map earth-like
share/snis/solarsystems/tau-scorpii/assets.txt:planet texture: erth-  no-normal-map earth-like
```

They should be "earthlike".  The diagnostics could be better, and caught earlier.

**Comment by MCMic** [comment]
Yeah but greping the whole git I also see:

```
snis_server.c:  else if (strcmp(solarsystem_assets->planet_type[planet_type], "earth-like") == 0)
infinite-taunt.c:       if (strcmp(s, "earth-like") == 0)
```

So there are two places in the code which refers to earth-like wording as well. (which is what made me doubt if maybe both keyword were existing, just in different places)

**Comment by MCMic** [comment]
Oh my bad your commit fixes all that

**Comment by smcameron** [comment]
Should be fixed by 2fbd1c6425af8eedf6dfa12a234de1339e49a99d plus I updated the files on spacenerdsinspace.com to fix the "earth-like" to be "earthlike".  You can "make update-assets" to get those few files, or grab the 500Mb archive: https://spacenerdsinspace.com/snis-asset-archives/snis_asset_archive_2019-01-26.tar.bz2

**Comment by MCMic** [comment]
Thanks!
